g/Generex/Generex.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Pl. add the header comments. Refer build-scripts repos for any of the scripts to get the standard comments.